common/app/views/support/Profile.scala

```diff
@@ -69,6 +69,7 @@ object Item620 extends Profile(width = Some(620))
 object Item640 extends Profile(width = Some(640))
 object Item700 extends Profile(width = Some(700))
 object Video640 extends VideoProfile(width = Some(640), height = Some(360)) // 16:9
+object FacebookOpenGraphImage extends Profile(width = Some(1200), height = Some(630))
```

Comment:
I don't think you should specify the height. What happens if the image is in a different aspect ratio to "normal".

I would just go with the width and preserve aspect ratio.

Other than that :+1: